README.md

```diff
@@ -1,2 +1,48 @@
 # nazareth-events
-A web app for viewing events in Nazareth
+A web app for viewing events in Nazareth.
+
+See the initial [prototype](http://nav-events.herokuapp.com/).
```

Comment:
this link is to an old prototype, either remove or change it to the link to the MVP